### PR TITLE
chore: GCP add account initialization doc

### PR DESCRIPTION
**Pull request by benjaminch**

### CODE REVIEW

File: website/docs/getting-started/install-qovery/gcp/cluster-managed-by-qovery/initialize-your-cloud-account-to-deploy-GKE.md.erb [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+That's it!
+
+</li>
+</ol>
+</Steps>
+
+Well done!! Your account is now ready to deploy GKE clusters.
+
+### Install a new cluster on Qovery
```
Comment (jul-dan) [inline website/docs/getting-started/install-qovery/gcp/cluster-managed-by-qovery/initialize-your-cloud-account-to-deploy-GKE.md.erb:83]: I would remove this part, as it is already specified in the quickstart guide

File: website/docs/getting-started/install-qovery/gcp/cluster-managed-by-qovery/quickstart.md.erb [hunk 2]
```diff
@@ -22,6 +22,24 @@ Install Qovery on your GCP account in less than 20 minutes. Qovery will create a
...
 <li>
 
+## Initialize your Google account
+
+Follow this [guide][docs.getting-started.install-qovery.gcp.cluster-managed-by-qovery.initialize-your-cloud-account-to-deploy-GKE] to initialize yout Google account allowing to deploy GKE clusters.
+Following GCP services should be enabled:
+* Kubernetes Engine API
+* Compute Engine API
+* Artifact Registry API
++ Cloud Storage API
```
Comment (jul-dan) [inline website/docs/getting-started/install-qovery/gcp/cluster-managed-by-qovery/quickstart.md.erb:32]:
```suggestion
* Cloud Storage API
```

**Review by jul-dan — APPROVED** [review]
LGTM just 2 minor fixes

File: website/docs/getting-started/install-qovery/gcp/cluster-managed-by-qovery/quickstart.md.erb (hunk 2 above)
Comment (benjaminch) [reply website/docs/getting-started/install-qovery/gcp/cluster-managed-by-qovery/quickstart.md.erb:32]: Good catch !

**Pull request merged**